### ISSUE TITLE
Can't send emails out of the domain

### ISSUE THREAD

**Issue by KireinaHoro** [mailcow/mailcow-dockerized#254]
Well, I can receive mails (locally and remotely - from Gmail) and send mails locally, but not remotely, say, none of my mails would reach an address outside the domain. Not even to zu49V8GiZXlNtD@dkimvalidator.com for DKIM validation...

I've set up DKIM and in domain settings it seems to be good and valid. SPF record reads "v=spf1 mx ~all".

**Comment by broedli** [comment]
What do the logs say when you try to send out a mail?

```
docker-compose logs --tail=100 postfix-mailcow
```

**Comment by KireinaHoro** [comment]
```
Attaching to mailcowdockerized_postfix-mailcow_1
[36mpostfix-mailcow_1    |[0m May  6 20:28:37 mail postfix/smtp[859]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:28:37 mail postfix/smtp[859]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 20:29:07 mail postfix/smtp[859]: connect to alt2.gmail-smtp-in.l.google.com[74.125.201.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:29:07 mail postfix/smtp[859]: 489E7186E99C: to=<xubin990510@gmail.com>, relay=none, delay=13960, delays=13870/0.04/91/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[74.125.201.27]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 20:37:37 mail postfix/qmgr[325]: 2DC65186E9E4: from=<i@jsteward.moe>, size=1142, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 20:38:08 mail enforced-tls-smtp/smtp[873]: connect to mail-tester.com[94.23.206.89]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:38:08 mail enforced-tls-smtp/smtp[873]: 2DC65186E9E4: to=<web-vodz9@mail-tester.com>, relay=none, delay=11707, delays=11675/0.06/31/0, dsn=4.4.1, status=deferred (connect to mail-tester.com[94.23.206.89]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 20:52:37 mail postfix/qmgr[325]: 90AD9186E9E3: from=<i@jsteward.moe>, size=1697, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 20:53:08 mail enforced-tls-smtp/smtp[886]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:53:08 mail enforced-tls-smtp/smtp[886]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 20:53:38 mail enforced-tls-smtp/smtp[886]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:53:38 mail enforced-tls-smtp/smtp[886]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 20:53:38 mail enforced-tls-smtp/smtp[886]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 20:53:38 mail enforced-tls-smtp/smtp[886]: 90AD9186E9E3: to=<xubin990510@gmail.com>, relay=none, delay=13431, delays=13370/0.04/61/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 20:57:37 mail postfix/qmgr[325]: 5316A186E993: from=<i@jsteward.moe>, size=1152, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 20:57:37 mail postfix/qmgr[325]: 1C68C186E9A3: from=<i@jsteward.moe>, size=1131, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 20:58:07 mail enforced-tls-smtp/smtp[899]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:58:07 mail enforced-tls-smtp/smtp[899]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 20:58:07 mail enforced-tls-smtp/smtp[899]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 20:58:09 mail enforced-tls-smtp/smtp[900]: connect to 30047900.in1.mandrillapp.com[52.35.107.200]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:58:37 mail enforced-tls-smtp/smtp[899]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:58:39 mail enforced-tls-smtp/smtp[900]: connect to 30047900.in1.mandrillapp.com[52.36.64.127]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:59:07 mail enforced-tls-smtp/smtp[899]: connect to alt2.gmail-smtp-in.l.google.com[74.125.201.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:59:07 mail enforced-tls-smtp/smtp[899]: 5316A186E993: to=<xubin990510@gmail.com>, relay=none, delay=15918, delays=15827/0.04/90/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[74.125.201.27]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 20:59:09 mail enforced-tls-smtp/smtp[900]: connect to 30047900.in1.mandrillapp.com[52.35.209.90]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 20:59:39 mail enforced-tls-smtp/smtp[900]: connect to 30047900.in2.mandrillapp.com[52.36.64.127]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:00:09 mail enforced-tls-smtp/smtp[900]: connect to 30047900.in2.mandrillapp.com[52.35.209.90]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:00:09 mail enforced-tls-smtp/smtp[900]: 1C68C186E9A3: to=<zu49V8GiZXlNtD@dkimvalidator.com>, relay=none, delay=12512, delays=12360/0.04/152/0, dsn=4.4.1, status=deferred (connect to 30047900.in2.mandrillapp.com[52.35.209.90]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 21:02:37 mail postfix/qmgr[325]: 489E7186E99C: from=<>, size=4782, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:02:38 mail postfix/smtp[919]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:03:08 mail postfix/smtp[919]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:03:38 mail postfix/smtp[919]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:03:38 mail postfix/smtp[919]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:03:38 mail postfix/smtp[919]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:03:38 mail postfix/smtp[919]: 489E7186E99C: to=<xubin990510@gmail.com>, relay=none, delay=16031, delays=15970/0.04/61/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1a]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 21:12:37 mail postfix/qmgr[325]: 2DC65186E9E4: from=<i@jsteward.moe>, size=1142, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:13:09 mail enforced-tls-smtp/smtp[932]: connect to mail-tester.com[94.23.206.89]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:13:09 mail enforced-tls-smtp/smtp[932]: 2DC65186E9E4: to=<web-vodz9@mail-tester.com>, relay=none, delay=13808, delays=13775/0.05/32/0, dsn=4.4.1, status=deferred (connect to mail-tester.com[94.23.206.89]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 21:27:37 mail postfix/qmgr[325]: 90AD9186E9E3: from=<i@jsteward.moe>, size=1697, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:28:08 mail enforced-tls-smtp/smtp[943]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:28:08 mail enforced-tls-smtp/smtp[943]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:28:08 mail enforced-tls-smtp/smtp[943]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:28:38 mail enforced-tls-smtp/smtp[943]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:28:38 mail enforced-tls-smtp/smtp[943]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:28:38 mail enforced-tls-smtp/smtp[943]: 90AD9186E9E3: to=<xubin990510@gmail.com>, relay=none, delay=15531, delays=15470/0.05/61/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 21:32:37 mail postfix/qmgr[325]: 5316A186E993: from=<i@jsteward.moe>, size=1152, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:32:37 mail postfix/qmgr[325]: 1C68C186E9A3: from=<i@jsteward.moe>, size=1131, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:33:07 mail enforced-tls-smtp/smtp[956]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:33:07 mail enforced-tls-smtp/smtp[956]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:33:08 mail enforced-tls-smtp/smtp[957]: connect to 30047900.in1.mandrillapp.com[52.35.209.90]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:33:37 mail enforced-tls-smtp/smtp[956]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:33:37 mail enforced-tls-smtp/smtp[956]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:33:37 mail enforced-tls-smtp/smtp[956]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:33:37 mail enforced-tls-smtp/smtp[956]: 5316A186E993: to=<xubin990510@gmail.com>, relay=none, delay=17988, delays=17927/0.05/60/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 21:33:38 mail enforced-tls-smtp/smtp[957]: connect to 30047900.in1.mandrillapp.com[52.36.64.127]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:34:08 mail enforced-tls-smtp/smtp[957]: connect to 30047900.in1.mandrillapp.com[52.35.107.200]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:34:38 mail enforced-tls-smtp/smtp[957]: connect to 30047900.in2.mandrillapp.com[52.36.64.127]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:35:08 mail enforced-tls-smtp/smtp[957]: connect to 30047900.in2.mandrillapp.com[52.35.209.90]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:35:08 mail enforced-tls-smtp/smtp[957]: 1C68C186E9A3: to=<zu49V8GiZXlNtD@dkimvalidator.com>, relay=none, delay=14611, delays=14460/0.04/151/0, dsn=4.4.1, status=deferred (connect to 30047900.in2.mandrillapp.com[52.35.209.90]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 21:37:37 mail postfix/qmgr[325]: 489E7186E99C: from=<>, size=4782, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:37:37 mail postfix/smtp[975]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:38:07 mail postfix/smtp[975]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:38:37 mail postfix/smtp[975]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:38:37 mail postfix/smtp[975]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:38:37 mail postfix/smtp[975]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 21:38:37 mail postfix/smtp[975]: 489E7186E99C: to=<xubin990510@gmail.com>, relay=none, delay=18130, delays=18070/0.04/61/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 21:47:37 mail postfix/qmgr[325]: 2DC65186E9E4: from=<i@jsteward.moe>, size=1142, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 21:48:08 mail enforced-tls-smtp/smtp[988]: connect to mail-tester.com[94.23.206.89]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 21:48:08 mail enforced-tls-smtp/smtp[988]: 2DC65186E9E4: to=<web-vodz9@mail-tester.com>, relay=none, delay=15906, delays=15875/0.04/31/0, dsn=4.4.1, status=deferred (connect to mail-tester.com[94.23.206.89]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 22:02:37 mail postfix/qmgr[325]: 90AD9186E9E3: from=<i@jsteward.moe>, size=1697, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 22:02:38 mail enforced-tls-smtp/smtp[999]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:03:08 mail enforced-tls-smtp/smtp[999]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:03:38 mail enforced-tls-smtp/smtp[999]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:03:38 mail enforced-tls-smtp/smtp[999]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:03:38 mail enforced-tls-smtp/smtp[999]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:03:38 mail enforced-tls-smtp/smtp[999]: 90AD9186E9E3: to=<xubin990510@gmail.com>, relay=none, delay=17631, delays=17570/0.05/61/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 22:07:37 mail postfix/qmgr[325]: 5316A186E993: from=<i@jsteward.moe>, size=1152, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 22:07:37 mail postfix/qmgr[325]: 1C68C186E9A3: from=<i@jsteward.moe>, size=1131, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 22:08:07 mail enforced-tls-smtp/smtp[1012]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:08:07 mail enforced-tls-smtp/smtp[1012]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:08:07 mail enforced-tls-smtp/smtp[1012]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:08:08 mail enforced-tls-smtp/smtp[1013]: connect to 30047900.in1.mandrillapp.com[52.35.209.90]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:08:37 mail enforced-tls-smtp/smtp[1012]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:08:37 mail enforced-tls-smtp/smtp[1012]: connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:08:37 mail enforced-tls-smtp/smtp[1012]: 5316A186E993: to=<xubin990510@gmail.com>, relay=none, delay=20088, delays=20027/0.05/60/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2607:f8b0:4001:c01::1b]:25: Cannot assign requested address)
[36mpostfix-mailcow_1    |[0m May  6 22:08:38 mail enforced-tls-smtp/smtp[1013]: connect to 30047900.in1.mandrillapp.com[52.35.107.200]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:09:08 mail enforced-tls-smtp/smtp[1013]: connect to 30047900.in1.mandrillapp.com[52.36.64.127]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:09:38 mail enforced-tls-smtp/smtp[1013]: connect to 30047900.in2.mandrillapp.com[52.35.209.90]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:10:08 mail enforced-tls-smtp/smtp[1013]: connect to 30047900.in2.mandrillapp.com[52.36.64.127]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:10:08 mail enforced-tls-smtp/smtp[1013]: 1C68C186E9A3: to=<zu49V8GiZXlNtD@dkimvalidator.com>, relay=none, delay=16712, delays=16560/0.04/151/0, dsn=4.4.1, status=deferred (connect to 30047900.in2.mandrillapp.com[52.36.64.127]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 22:12:37 mail postfix/qmgr[325]: 489E7186E99C: from=<>, size=4782, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 22:13:07 mail postfix/smtp[1030]: connect to gmail-smtp-in.l.google.com[108.177.97.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:13:07 mail postfix/smtp[1030]: connect to gmail-smtp-in.l.google.com[2404:6800:4008:c00::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:13:37 mail postfix/smtp[1030]: connect to alt1.gmail-smtp-in.l.google.com[74.125.30.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:13:37 mail postfix/smtp[1030]: connect to alt1.gmail-smtp-in.l.google.com[2607:f8b0:4003:c03::1a]:25: Cannot assign requested address
[36mpostfix-mailcow_1    |[0m May  6 22:14:07 mail postfix/smtp[1030]: connect to alt2.gmail-smtp-in.l.google.com[74.125.201.27]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:14:07 mail postfix/smtp[1030]: 489E7186E99C: to=<xubin990510@gmail.com>, relay=none, delay=20261, delays=20170/0.14/91/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[74.125.201.27]:25: Connection timed out)
[36mpostfix-mailcow_1    |[0m May  6 22:22:37 mail postfix/qmgr[325]: 2DC65186E9E4: from=<i@jsteward.moe>, size=1142, nrcpt=1 (queue active)
[36mpostfix-mailcow_1    |[0m May  6 22:23:08 mail enforced-tls-smtp/smtp[1046]: connect to mail-tester.com[94.23.206.89]:25: Connection timed out
[36mpostfix-mailcow_1    |[0m May  6 22:23:08 mail enforced-tls-smtp/smtp[1046]: 2DC65186E9E4: to=<web-vodz9@mail-tester.com>, relay=none, delay=18006, delays=17975/0.06/31/0, dsn=4.4.1, status=deferred (connect to mail-tester.com[94.23.206.89]:25: Connection timed out)
```

**Comment by KireinaHoro** [comment]
Ugh.. this seems to be a connection problem. I can't connect to the respective servers on port 25 via telnet either. (`alt1.gmail-smtp-in.l.google.com` for example)

Yet the hosts do ping from my server. My firewall is permissive of all outbound connections.

**Comment by KireinaHoro** [comment]
Ah... I got it.

My servers are hosted on Vultr, and they block SMTP by default. I'll try opening a ticket to complain about this.

**Comment by broedli** [comment]
I would guess that your provider blocks connections on port 25.

~~It's generally a bad idea to run a mailserver from a home / consumer connection.~~

Edit: Lol, I'm too tired - going to bed now!

**Comment by KireinaHoro** [comment]
Thanks a lot. Problem resolved!

**Comment by regalstreak** [comment]
@KireinaHoro do they open the ports on opening a ticket?

**Comment by benjif** [comment]
This issue is long closed, but for future searchers: Vultr will indeed open the SMTP port upon request, depending on what you're using the email service for (they ask).

**Comment by andryyy** [comment]
Cool, thank you. :)

**Comment by donniecode1983** [comment]
> Ah... I got it.
> 
> My servers are hosted on Vultr, and they block SMTP by default. I'll try opening a ticket to complain about this.

Thanks for this! I ran into the same problem with Vultr, had no clue it was them until i ran across this post.

**Comment by mmhy2003** [comment]
I know it is already closed issue, is there a way to configure postfix to use TLS or SSL by default instead because it seems all cloud providers block port 25.

**Comment by andryyy** [comment]
Port 25 is the defined standard to listen for external senders. Almost no server will accept mail on another port and if it does, you don’t know what happens to this mail, as it might be a misconfiguration by the admin.

Port 25 is luckily mostly blocked by cloud providers and will only be unblocked after identifying yourself and declaring what you are up to.

Literally everybody is sick of spam mails, it’s a stupid business for stupid people.

If your intentions are fine and you won’t send hundreds of mails to people who did not agree to read your stuff, you will be granted access to the holy mailing universe.

**Comment by NGOcoins** [comment]
so there is NO way to send emails via 587?!
